CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
+
+0.100.0 Release notes (2016-05-02)
+=============================================================
+* Updated to Realm Objective-C 0.100.0
```

Comment:
It'd be worth mentioning that this update prevents corrupting encrypted Realm files that were written using `-writeCopyToURL:encryptionKey:error:`.